django/forms/fields.py

```diff
@@ -204,6 +204,19 @@ def has_changed(self, initial, data):
         data_value = data if data is not None else ''
         return initial_value != data_value
 
+    def get_bound_field(self, form, name):
```

Comment:
I remember why I didn't call it that in the first place: The call to `Field.get_bound_field()` happens in `Form.__getitem__()` and there it is called `name`. It's probably still better naming it `field_name` in `get_bound_field()`, though.